test/functional/p2p_filter.py

```diff
@@ -153,6 +184,33 @@ def run_test(self):
         filter_node.send_and_ping(msg_filterload(data=b'', nHashFuncs=1))
         filter_node.send_and_ping(msg_filteradd(data=b'letstrytocrashthisnode'))
 
+    def run_test(self):
+        filter_node = self.nodes[0].add_p2p_connection(FilterNode())
```

Comment:
nit: one style point that I've tried to move the tests away from is referring to connections as "nodes". A node is a bitcoind instance that we're testing. A p2p connection is a test structure used to interact with the node. Using the word _node_ for both the node and connections to it is confusing.

Reply:
Noted! Renaming all of them increases the number of lines in the diff by a lot, so I put renames for existing code in a separate commit.